_guides/https.md

```diff
@@ -7,24 +7,33 @@ description: Easily configure The Lounge to be served over HTTPS for better secu
 In this guide, we will see how to easily configure The Lounge to be served over [HTTPS](https://en.wikipedia.org/wiki/HTTPS) for better security and privacy.
 
 {: .alert.alert-warning role="alert"}
-The Lounge only has basic HTTPS support, and will need to be manually restarted to reload certificates on renewal. For advanced HTTPS support, consider [using a reverse proxy](/docs/guides/reverse-proxies).
+The Lounge only has basic HTTPS support.
 
-First, you need an HTTPS certificate. [Let's Encrypt](https://letsencrypt.org/) is a free, automated, and open Certificate Authority that provides completely free HTTPS certificates.
+First, you need an HTTPS certificate and a domain. Make sure your domain's A record pointing to your server's IP. [Let's Encrypt](https://letsencrypt.org/) is a free, automated, and open Certificate Authority that provides completely free HTTPS certificates.
 
-Assuming you have a valid email address at `email@example.com`, and want to serve The Lounge at `https://thelounge.example.com`, run these commands on your server:
+Assuming your domain is `thelounge.example.com`, follow these steps:
 
+1. Install nginx with `sudo apt install nginx`.
+2. Create configuration file for your domain with `sudo nano /etc/nginx/sites-available/thelounge.example.com`
+3. Paste code below:
 ```
```

Comment:
This is not an improvement over the reverse proxy docs, you are essentially writing an opinionated guide that *only* works on Debian / Ubuntu specific distros.

`/etc/nginx/sites-enabled/` doesn't do anything if you don't have nginx setup to include those configs.  
`/etc/nginx/sites-available` is just an admin convention, nginx doesn't care about it.